### PR TITLE
LPS-182078 | Test Automation | Master

### PR DESCRIPTION
**Pull request by davisaints**
**JIRA Ticket: [LPS-182078](https://issues.liferay.com/browse/LPS-182078)**

### CODE REVIEW

**Comment by davisaints** [comment]
ci:test:sf

**Comment by andreefarias** [comment]
This testcase is blocked by the bug: [LPS-181920](https://issues.liferay.com/browse/LPS-181920).

Waiting for merge to review it.

**Review by andreefarias — APPROVED** [review]
Nice @davisaints, good job!

LGTM! 🎉

**Comment by andreefarias** [comment]
ci:test:sf

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectFields.testcase [hunk 1]
```diff
@@ -177,6 +177,55 @@ definition {
...
+				fieldType = "Long",
+				fileSource = "userComputer",
+				isRequired = "false",
+				objectName = "CustomObject182078",
+				showFilesInDocumentsAndMedia = "true");
+
+			ObjectAdmin.publishObjectViaAPI(objectName = "CustomObject182078");
+		}
+
+		task ("And given a custom object entry") {
```
Comment (timpak) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectFields.testcase:204]: And given an object entry with an attachment

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectFields.testcase [hunk 2]
```diff
@@ -177,6 +177,55 @@ definition {
...
+			ObjectAdmin.goToCustomObject(objectName = "CustomObject182078");
+
+			ObjectAutoGenerated.addAttachment(fileName = "LiferayLogo.png");
+
+			PortletEntry.save();
+		}
+
+		task ("When going to Global Site and into the Documents and Media") {
```
Comment (timpak) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectFields.testcase:212]: When the user navigates to Documents and Media in the global site

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectFields.testcase [hunk 3]
```diff
@@ -177,6 +177,55 @@ definition {
...
+			ApplicationsMenu.gotoSite(site = "Global");
+
+			ProductMenu.gotoPortlet(
+				category = "Content & Data",
+				portlet = "Documents and Media");
+		}
+
+		task ("Then assert that exist the attachment field from a custom object") {
```
Comment (timpak) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectFields.testcase:220]: Then a folder is seen that contains the attachment from the object entry

**Review by timpak — CHANGES_REQUESTED** [review]
@davisaints good job with this test! Can you take a look at my comments below? You can forward this once you make the changes.

**Comment by davisaints** [comment]
ci:test:sf

**Comment by davisaints** [comment]
ci:forward

**Comment by davisaints** [comment]
ci:reopen

**Comment by davisaints** [comment]
ci:forward

**Pull request closed without merging**